prospero-cli/src/main/java/org/wildfly/prospero/cli/commands/AbstractCommand.java

```diff
@@ -23,4 +29,24 @@ public AbstractCommand(Console console, ActionFactory actionFactory) {
         this.console = console;
         this.actionFactory = actionFactory;
     }
+
+    static Path determineInstallationDirectory(Optional<Path> directoryOption) {
```

Comment:
do the new methods need to be static?

Reply:
Not necessarily, I like to mark the methods static if they are not dependent on the instances' inner state and don't manipulate the inner state, to highlight that fact. But it's matter of taste... Let me know if you still prefer non-static :].

Reply:
Fair enough. I tend to try and avoid static methods when possible, but like you say it's matter of taste.
Happy to leave that in